### ISSUE TITLE
False positive for commented code

### ISSUE THREAD

**Issue by smichaelsen** [martin-helmich/typo3-typoscript-lint#40]
The following comment in our TypoScript raises a commented code info:

````
# Processing <ol>, <ul> and <table> blocks separately
````

I guess because `<` and `>` are recognized as TypoScript operators. Is there a way to fix this?

**Comment by sascha-egerer** [comment]
I don't think it has anything todo with `<` or `>?. I do also get that warning for a line like `# Overwrite staging specific values`
